examples/poetry_example/Dockerfile

```diff
@@ -0,0 +1,15 @@
+##  Could be any base image, only build dependencies are git and pip
+FROM python:3.8-slim
+
+ENV PROJECT_NAME='commands-base-example'
+
+RUN apt-get update && apt-get install -y git && \
+    pip install poetry==1.0.5
+WORKDIR /the/workdir/path
+ENV ALADDIN_COMMANDS_PATH='/the/workdir/path/commands_base_example'
+
+COPY pyproject.toml ./
```

Comment:
added it as a comment, I did not want to commit that file here because it's basically a duplicate of the lock file in the repo root? which also makes this example harder to maintain, in any case I don't think we're aiming for this to be a poetry how-to

Reply:
Can't we commit a dummy file here with a comment in it saying that it's a placeholder?

Reply:
hmm so I don't see how a dummy file is an improvement over the comment in the line below, if using a blank dummy file with just a comment in it then the example is broken because poetry will complain about the lock file being out of sync with .toml, this way at least the example is pretty much ready to go, in any case I don't think we're aiming for this to be a poetry how-to